target/linux/x86/config-5.4

```diff
@@ -37,7 +37,7 @@ CONFIG_ARCH_RANDOM=y
 CONFIG_ARCH_SELECT_MEMORY_MODEL=y
 CONFIG_ARCH_SPARSEMEM_ENABLE=y
 CONFIG_ARCH_STACKWALK=y
-CONFIG_ARCH_SUPPORTS_ACPI=y
+ONFIG_ARCH_SUPPORTS_ACPI=y
```

Comment:
we shouldn't be modifying the target linux kernel configs - this will affect all other targets, which may not need dpdk/bess related modules enabled 

any linux target configs should be done through the KCONFIG options per package